src/luxonis_ml/data/dataset.py

```diff
@@ -638,7 +637,7 @@ def _add_extract(self, additions, from_bucket):
                         Key=new_prefix,
                         CopySource={'Bucket': self.bucket, 'Key': old_prefix}
                     )
-                else:
+                elif filepath != "{local_cache}/{component_name}/{granule}":
```

Comment:
@tersekmatija I wonder if we can use `media_exists` here instead or find the cause of the `cp` errors since this seems like a workaround.

Actually, maybe we should even remove the `media_exists` entirely and only upload/extract the data in the case that the unique hash file does not exist. We can use `os` or `boto3` for this.